### PR TITLE
ID-Console Suspension instead of Termination

### PR DESCRIPTION
**Pull request by Arrow768**
The ID Console now suspends employees instead of terminating them
https://forums.aurorastation.org/viewtopic.php?f=18&t=9785

### CODE REVIEW

File: code/modules/modular_computers/file_system/programs/command/card.dm [hunk 1]
```diff
@@ -163,11 +163,11 @@
 				if(id_card)
 					data_core.manifest_modify(id_card.registered_name, id_card.assignment)
 				computer.proc_eject_id(user)
-		if("terminate")
+		if("suspend")
 			if(computer && can_run(user, 1))
-				id_card.assignment = "Terminated"
+				id_card.assignment = "Suspendet"
```
Comment (Lohikar) [inline code/modules/modular_computers/file_system/programs/command/card.dm]: Spelling.

File: code/controllers/hooks-defs.dm [hunk 2]
```diff
@@ -77,7 +77,7 @@
  * Called in card.dm when someone's card is terminated at the HoP's desk.
  * Parameters: var/obj/item/weapon/card/id
  */
-/hook/terminate_employee
+/hook/suspend_employee
```
Comment (Lohikar) [inline code/controllers/hooks-defs.dm]: Please update the comment too.

File: nano/templates/identification_computer.tmpl [hunk 3]
```diff
@@ -55,10 +55,10 @@
 
 	<div class='item'>
 	  <div class='itemLabel'>
-		Terminations:
+		Suspensions:
 	  </div>
 	  <div class='itemContent'>
-		{{:helper.link('Terminate ' + data.id_owner, 'gear', {'action' : 'terminate'}, data.id_rank == "Terminated" ? 'disabled' : null, data.id_rank == "Terminated" ? 'disabled' : 'linkDanger')}}
+		{{:helper.link('Suspend ' + data.id_owner, 'gear', {'action' : 'suspend'}, data.id_rank == "Suspendet" ? 'disabled' : null, data.id_rank == "Suspendet" ? 'disabled' : 'linkDanger')}}
```
Comment (Lohikar) [inline nano/templates/identification_computer.tmpl]: Here too.

**Comment by Arrow768** [comment]
And its done

**Pull request merged**